### PR TITLE
Fix ready-for-merge tests that fail because of file encoding problems

### PR DESCRIPTION
**Pull request by lptr**
Fixes https://github.com/gradle/gradle/issues/25429

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test RfR

**Comment by lptr** [comment]
@bot-gradle cancel

**Comment by lptr** [comment]
@bot-gradle test RfR

File: subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/bundling/JarEncodingIntegrationTest.groovy [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+import org.gradle.test.fixtures.file.DoesNotSupportNonAsciiPaths
+import spock.lang.Issue
+
+import java.util.jar.JarFile
+import java.util.jar.Manifest
+
+@TestReproducibleArchives
+@DoesNotSupportNonAsciiPaths(reason = "Tests manage their own encoding settings")
+class JarEncodingIntegrationTest extends AbstractIntegrationSpec {
+    // Only works on Java 8, see https://bugs.openjdk.java.net/browse/JDK-7050570
```
Comment (tresat) [inline subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/bundling/JarEncodingIntegrationTest.groovy:32]: ❔ Is this comment still true?  Doesn't think work on Java >= 8?

File: subprojects/core/src/integTest/groovy/org/gradle/api/AbstractHttpScriptPluginIntegrationSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+package org.gradle.api
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.TestResources
+import org.gradle.test.fixtures.keystore.TestKeyStore
+import org.gradle.test.fixtures.server.http.HttpServer
+import org.gradle.test.matchers.UserAgentMatcher
+import org.gradle.util.GradleVersion
+
+class AbstractHttpScriptPluginIntegrationSpec extends AbstractIntegrationSpec {
```
Comment (tresat) [inline subprojects/core/src/integTest/groovy/org/gradle/api/AbstractHttpScriptPluginIntegrationSpec.groovy:26]: 🧐 Shouldn't this class be `abstract`?

**Review by tresat — APPROVED** [review]

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Comment by ljacomet** [comment]
Went ahead and sent it for merge since it will unblock RfR

**Pull request merged**